pkg/app/server/grpcapi/web_api.go

```diff
@@ -121,7 +118,7 @@ type WebAPI struct {
 	pipedProjectCache      cache.Cache
 	pipedStatCache         cache.Cache
 	insightCache           cache.Cache
-	redis                  redis.Redis
+	unregisteredAppStore   unregisteredappstore.Store
```

Comment:
nits, should move this to the upper block (nearby other stores) for better readable code 👍  (same with the piped_api.go PipedApi struct)